service/pom.xml

```diff
@@ -64,6 +70,11 @@
             <artifactId>httpclient5</artifactId>
             <version>5.4</version>
```

Comment:
das bitte bei der Gelegenheit auch umziehen...wie ist das denn reingekommen? Ich dachte ich hätte alle Versionen rausgeschmissen

Reply:
I suspect that was in there because if you move it to the root POM the version clashes with `5.3.1` which is used by spring-web. I'm not entirely sure how we can resolve this. A quick fix would be to set this to `5.3.1` but that would break on the next Spring update...